jellyfin-api-ktor/src/jvmMain/kotlin/org/jellyfin/sdk/api/ktor/KtorClient.kt

```diff
@@ -136,7 +136,7 @@ public actual open class KtorClient actual constructor(
 			throw InvalidContentException("Serialization failed", err)
 		} catch (err: SSLException) {
 			logger.error(err) { "Unknown SSL error occurred" }
-			throw SecureConnectionException("Unknown SSL error occurred", err)
+			throw SSLExceptionResolver.resolveSSLException(err)
```

Comment:
This can be inlined (e.g. the when cases from SSLExceptionResolver can be added here)